docs/operating-scylla/procedures/cluster-management/rebuild-node.rst

```diff
@@ -1,19 +1,25 @@
 ============================================
-Rebuild a Node After Losing the Data Volume
+Recover a Node After Losing the Data Volume
 ============================================
 
-When running in Scylla on EC2, it is recommended to use i3 type instances, storing the data on fast, ephemeral SSD drives.
-Stopping the node and starting it for whatever reason means the data on the node is lost.
-This data loss applies not only to i3 type instances but to the i2 type as well.
-The good news is Scylla is a HA database, and replicas of the data are stored on additional nodes.
-This is also why it's highly recommended to use a replication factor of at least three per Data Center (for example, 1 DC, RF = 3, 2 DCs RF = 6).
+When running in Scylla on EC2, it is recommended to use instances with fast, ephemeral SSD drives.
+Stopping and starting the instance for whatever reason means the data on the node is lost.
 
-To recover the data and rebuild the node, follow this procedure:
+The good news is that ScyllaDB is an HA database, and replicas of the data are stored on multiple nodes. 
+This is also why it's highly recommended to use a replication factor of at least three per Data Center, for example, ``replication = {'class': 'NetworkTopologyStrategy', 'DC1' : 3, 'DC2' : 3}``.
+
+The following procedure is a particular use case of :doc:`Replace a Dead Node in a ScyllaDB Cluster Procedure </operating-scylla/procedures/cluster-management/replace-dead-node>`, when a node replaces itself using its previous identity (Host ID) and IP address.
 
 #. Open the ``/etc/scylla/scylla.yaml`` file.
 
-#. Add, if not present, else edit, the ``replace_node_first_boot`` parameter and change it to the
+#. Add, if not present, else edit the ``replace_node_first_boot`` parameter and change it to the
    Host ID of the node before it restarted.
+
+#. If you know of dead nodes in the cluster, for example, from :doc:`nodetool status </operating-scylla/nodetool-commands/status>`, use :doc:`ignore_dead_nodes_for_replace </reference/configuration-parameters>` to list them.
```

Comment:
Can we add an anchor to `ignore_dead_nodes_for_replace` in the link?
So to point to http://previews.docs.scylladb.com/scylladb/17645/reference/configuration-parameters/#confval-ignore_dead_nodes_for_replace

Reply:
Ah, it's actually written below as a TODO item.
What prevents it? (I'm concerned that we'll never get to it if we don't do it now)

Reply:
It just got merged, 3 hours ago :)

Reply:
This is an endless cycle...
I will close this PR and send a new fix one at a later date.